rclc_lifecycle/README.md

```diff
@@ -26,7 +26,7 @@ rcl_node_t my_node = rcl_get_zero_initialized_node();
 rc = rclc_node_init_default(&my_node, "lifecycle_node", "rclc", &support);
```

Comment:
```
rcl_node_t my_node;
```
`rcl_get_zero_initialized_node()` is now called in `rclc_node_init_xxx` function. You can remove that at the declaration. Also in the example source code (for server and client).